ogr/ogrsf_frmts/gpkg/ogrgeopackagedatasource.cpp

```diff
@@ -10153,8 +10146,8 @@ bool GDALGeoPackageDataset::AddFieldDomain(
             if (sqlite_err != SQLITE_OK && sqlite_err != SQLITE_DONE)
             {
                 CPLError(CE_Failure, CPLE_AppDefined,
-                         "failed to execute insertion: %s",
-                         sqlite3_errmsg(hDB));
+                         "failed to execute insertion '%s': %s",
+                         sqlite3_errmsg(hDB), sqlite3_errmsg(hDB));
```

Comment:
Is this double up intended?

Reply:
good catch. fixed in d8530eb3b004271b77566e5c6fee30e37712a37d